lib/tasks/equity_composites.rake

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# Run rake db:seed:equity_composites to load the equity composite index data
+
+namespace :db do
+  namespace :seed do
+    desc "Load equity composite index data"
+    task equity_composites: :environment do
```

Comment:
ℹ️ I added a rake task both for us and staging. We might want one-time to load the equity data without blowing away our local db.